apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -270,6 +267,10 @@ export class LandingPage extends ViewPlugin {
       await this.appManager.activatePlugin(['solidity', 'mythx'])
       this.verticalIcons.select('mythx')
     }
+    const startLearnEth = async () => {
+      await this.appManager.activatePlugin(['solidity', 'LearnEth'])
```

Comment:
solidityUnitTesting (please double check the name) also need to be activated

Reply:
done